src/Task/Assets/loadTasks.php

```diff
@@ -1,41 +0,0 @@
-<?php
-namespace Robo\Task\Assets;
-
-trait loadTasks
-{
-    /**
-    * @param $input
-    * @return Minify
-    */
-    protected function taskMinify($input)
-    {
-        return new Minify($input);
```

Comment:
this can be rewritten as:

```
return $this->task('minify');
```

and what's important we have docblock which states the type of return object. SO that makes tasks loaded with this method to be written easily